html-report-generator/generate_usage_report.py

```diff
@@ -36,6 +33,13 @@ def main():
     with open(config_file_path, 'r') as file:
         dir_list = [line.strip() for line in file]
 
+
+    # Get the name of the filesystem from config.txt
+    filesystem_name = dir_list[0].split('/')[1]
+
+    # Title report file as report-data-{datestamp}.html or report-rfanfs-{datestamp}.html
+    file_path = f"{pjoin(working_dir, '_data/', 'htmlreport/')}report-{filesystem_name}-{current_date:%Y%m%d}.html"    
```

Comment:
Did you test it? `()` within `f"{ }"` do not usually work. Hence, we had to stick to `.format()`